src/js/common/components/NavButtons.jsx

```diff
@@ -165,7 +165,7 @@ export default class NavButtons extends React.Component {
     } else if (path.endsWith('submit-message')) {
       buttons = (
         <div className="row form-progress-buttons">
-          <div className="small-6 medium-5 columns">
+          <div className="small-12 medium-5 columns">
```

Comment:
I also changed this one because I didn't want it to be tiny on a small screen when it's the only button. Same problem as the "Get Started" button.